R/pkg/R/stats.R

```diff
@@ -19,7 +19,8 @@
 
 setOldClass("jobj")
 
-#' crosstab
+#' @title SparkDataFrame statistic functions
```

Comment:
Fixed, sorry about that.

![image](https://cloud.githubusercontent.com/assets/8969467/16218376/fecbe46e-3731-11e6-9b18-c17e72df46e6.png)

`cov`, `corr` shouldn't be there - they are referenced in generic.R these bugs are also fixed in my other PR #13798 - there are quite a lot of them.